src/main/java/org/broadinstitute/consent/http/mail/freemarker/TemplateModel.java

```diff
@@ -1,20 +1,13 @@
 package org.broadinstitute.consent.http.mail.freemarker;
 
+/* This model works for the Send Reminder Message template */
 public class TemplateModel {
```

Comment:
If this model is now only single-use, would it be worth renaming to something more specific?

Reply:
Great point. I struggle sometimes with the scope of changes. I'll bite the bullet and make it more specific. This class, by the way, goes back to the earliest days of the app ... the very first commit has this message:
```
/* This model works for every template we have: collect, new-case and reminder. The placeholders are all the same. */
```